snippets/lvms-scaling-up-storage-lvmcluster-cr-snippet.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: SNIPPET
+.Example `LVMCluster` CR
+[source,yaml]
+----
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: my-lvmcluster
+spec:
+  storage:
+    deviceClasses:
+# ...
+      deviceSelector: <1>
+        paths: <2>
+        - /dev/disk/by-path/pci-0000:87:00.0-nvme-1
+        - /dev/disk/by-path/pci-0000:88:00.0-nvme-1
+        optionalPaths: <3>
+        - /dev/disk/by-path/pci-0000:89:00.0-nvme-1
+        - /dev/disk/by-path/pci-0000:90:00.0-nvme-1
+# ...
+----
+<1> Contains the configuration to specify the paths to the devices that you want to add to the LVM volume group.
+You can specify the device paths in the `paths` field, the `optionalPaths` field, or both. If you do not specify the device paths in both `paths` and `optionalPaths`, {lvms} adds the supported unused devices to the LVM volume group. {lvms} adds the devices to the LVM volume group only if the following conditions are met:
```

Comment:
Same comment about the first mention of lvms here. However, it's tricky, because right now the snippet is included in modules that don't mention the term beforehand, but _technically_ it can be. If you don't anticipate including this snippet elsewhere then I'd still make this change. But either way I think it's fine to play it safe:
```suggestion
You can specify the device paths in the `paths` field, the `optionalPaths` field, or both. If you do not specify the device paths in both `paths` and `optionalPaths`, {lvms-first} adds the supported unused devices to the LVM volume group. {lvms} adds the devices to the LVM volume group only if the following conditions are met:
```

Reply:
Updated as per the suggestion.